spec/furniture/journal/entry_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe Journal::Entry do
+  subject(:entry) { build(:journal_entry, body: body) }
+
+  let(:body) { Faker::Books::CultureSeries.civ }
```

Comment:
I don't think you need this `body` declaration, it is never used (the one on line 12 is used instead).

Reply:
True!